python/init.sls

```diff
@@ -46,12 +46,17 @@ setuptools:
     - upgrade: True
     - require:
       - pkg: python-pkgs
+      - pip: pip
 
 pip:
   pip.installed:
+{% if grains['saltversion'] < '2016.3.2' %}
     - name: pip==8.1.1
+{% else %}
+    - upgrade: True
```

Comment:
Technically no, since if `name` is not specified, then salt uses the title of the state as the name. But I can change it to be more explicit.